lib/animina/user_email.ex

```diff
@@ -20,16 +20,32 @@ defmodule Animina.UserEmail do
       Ash.CiString.value(user.email),
       subject,
       text_body <>
-        "\n#{"https://animina.de/auth/user/confirm_new_user?#{URI.encode_query(confirm: confirm)}"}"
+        "\n<a href='#{"#{get_link(Application.get_env(:animina, :env))}/auth/user/confirm_new_user?#{URI.encode_query(confirm: confirm)}"}' target='_blank'>#{gettext("Confirm your email address")}</a>"
     )
   end
 
+  defp get_link(:prod) do
```

Comment:
This approach feels like reinventing the wheel. Let's use the config we already have in the system:

```elixir
  host = System.get_env("PHX_HOST") || "example.com"
  port = String.to_integer(System.get_env("PORT") || "4000")

...

  config :animina, AniminaWeb.Endpoint,
    url: [host: host, port: 443, scheme: "https"],
 ```

You can find it in `runtime.exs`.

Reply:
We just have to add a text to the README.md that those env variables have to be set.

Reply:
@wintermeyer  , kindly help me here .
In production would host be "animina.de" , also how would I use this value in runtime in my code for the conditional url link , I found this 
[Elixir Forum Link](https://elixirforum.com/t/how-to-get-runtime-config-during-runtime-exs/49236/3)  , but It did not.

I am yet to push anything to prod that needs different values than the one in dev using this format hence my questions , thanks

Reply:
Have a look at https://chatgpt.com/share/056c8924-dfbb-4ced-b5cf-096d5f2b2bbd

BUT: We do have bigger fish to fry right now. In case you can't get this running just create a new issue and kick the can down the road.

Reply:
Okay @wintermeyer  , let me move on to the long expiry time task for tokens then , you can close this PR

Reply:
@MICHAELMUNAVU83 Can I merge this PR or should I close it?

Reply:
Merge